library/src/lib/toggle/toggle.component.ts

```diff
@@ -19,49 +22,71 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 })
 export class ToggleComponent implements OnInit, ControlValueAccessor {
 
+    /** @Input The size of the toggle. Can be one of the four *xs*, *s*, *l*, *error* or default. */
     @Input()
     size: string;
 
+    /** @Input Whether the toggle is disabled. */
     @Input()
     disabled: boolean = false;
 
+    /** @Input Id for the toggle component. If omitted, a unique one is generated. */
     @Input()
     id: string;
 
+    /** @Input Whether the toggle is checked. */
     @Input()
     checked: boolean = false;
 
+    /** @Output Event fired when the toggle is checked. */
     @Output()
     checkedChange = new EventEmitter<boolean>();
 
-    onChange: any = () => {};
-    onTouched: any = () => {};
+    /** @hidden */
+    onChange: any = () => { };
 
+    /** @hidden */
+    onTouched: any = () => { };
+
+    /** Get the isChecked property of the toggle. */
     get isChecked() {
         return this.checked;
     }
 
+    /** Set the isChecked property of the toggle. */
     set isChecked(value) {
         this.checked = value;
         this.onChange(value);
         this.onTouched();
         this.checkedChange.emit(value);
     }
 
+    /**
+     * @param value Sets the value of the *checked* property of the toggle.
+     */
     writeValue(value: any) {
         this.checked = value;
     }
 
+    /**
+     * @param fn User defined function that handles the *onChange* event of the toggle.
+     */
     registerOnChange(fn) {
```

Comment:
Keep the docs, but add @hidden. In general, this should not be used be users. It is used by Angular forms to register a behaviour for onChange.